openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -11,10 +18,61 @@
     deserialize_bytes_to_event_data,
     serialize_event_data_to_bytes,
 )
+from openedx_events.learning import signals  # pylint: disable=reimported
+from openedx_events.tooling import OpenEdxPublicSignal
+
+
+def generate_test_event_data_for_signal(class_type):
```

Comment:
This parameter name is misleading. The argument is not really a type, it's an instance of a signal

Reply:
I call this function recursively for the classes contained within the signal, are the classes within the signals also themselves signals?